modules/getting-started/proc-customize-rhdh-learning-paths.adoc

```diff
@@ -0,0 +1,23 @@
+[id='proc-customize-rhdh-learning-paths_{context}']
+= Customizing the Learning Paths in {product}
+
+
+In {product}, you can configure Learning Paths by passing the data into the `app-config.yaml` file as a proxy. You can provide data to the Learning Path using the following sources:
+
+* JSON files hosted on GitHub or GitLab. To access the data from the JSON files, add the following code to the `app-config.yaml` file:
++
+[source,yaml]
+----
+proxy:
+  endpoints:
+    '/developer-hub':
+      target: https://raw.githubusercontent.com/
+      pathRewrite:
+        '^/api/proxy/developer-hub/learning-paths': '/janus-idp/backstage-showcase/main/packages/app/public/learning-paths/data.json'
+        '^/api/proxy/developer-hub/tech-radar': '/janus-idp/backstage-showcase/main/packages/app/public/tech-radar/data-default.json'
+        '^/api/proxy/developer-hub': '/janus-idp/backstage-showcase/main/packages/app/public/homepage/data.json'
+      changeOrigin: true
+      secure: true
+----
+
+//* A separate service that provides the Learning Path data in JSON format using an API.
```

Comment:
yeah, it'll be needed, `app-config-rhdh.yaml` content will be as below. 

can refer to the PR description as well https://github.com/janus-idp/backstage-showcase/pull/1184

```
   proxy:
      endpoints:
        # Other Proxies
        '/developer-hub/learning-paths':
          target: ${LEARNING_PATH_DATA_URL}
          changeOrigin: true
          # Change to "false" in case of using self hosted cluster with a self-signed certificate
          secure: true
```